css/index.css

```diff
@@ -233,52 +259,85 @@ header nav a:not(:first-child):hover {
     padding: 15px;
     background-color: #ffffff;
     font-size: 16px;
+	font-family: 'Preahvihear', sans-serif;
     filter: drop-shadow(2px 2px 2px rgba(0, 0, 0, 0.5));
     transition: background-color .4s linear;
 }
 
 .skills_list li:nth-child(3n+1) {
-    border-left: 4px solid rgba(240, 217, 123, 1.00);
+	border-left: 4px solid var(--yellow-color);
 }
+
 .skills_list li:nth-child(3n+1):hover {
-    background-color: rgba(240, 217, 123, 1.00);
+	background-color: var(--yellow-color);
 }
 
 .skills_list li:nth-child(3n+2) {
-    border-left: 4px solid rgba(252, 219, 214, 1.00);
+	border-left: 4px solid var(--pink-color);
 }
 
 .skills_list li:nth-child(3n+2):hover {
-    background-color: rgba(252, 219, 214, 1.00);
+	background-color: var(--pink-color);
 }
 
 
 .skills_list li:nth-child(3n+3) {
-    border-left: 4px solid rgba(196, 226, 226, 1.00);
+	border-left: 4px solid var(--blue-color);
 }
 
 .skills_list li:nth-child(3n+3):hover {
-    background-color: rgba(196, 226, 226, 1.00);
+	background-color: var(--blue-color);
 }
 
 
 /*********************** EXPERIENCE *****************/
 
 .experience {
-    background-color: rgba(229, 231, 233, 0.3);
+	background-color: rgba(229, 231, 233, 0.1);
 }
 
 .experience_inner {
-    padding: 20px;
-    background-color: rgba(196, 226, 226, 0.50);
+	padding: 20px;
+	background-color: rgba(196, 226, 226, 0.50);
 }
 
 .experience_header {
-    font-size: 28px;
-    font-family: 'Caveat Brush', cursive;
-    letter-spacing: 2px;
-    text-transform: uppercase;
-    line-height: 1.2;
+	margin: 10px 0;
+	text-transform: uppercase;
+	text-align: center;
+	font-size: 28px;
+	font-family: 'Caveat Brush', cursive;
+	letter-spacing: 2px;
+	line-height: 1.2;
+}
+
+.experience_list {
+	padding: 0;
+	display: flex;
+	justify-content: center;
+}
+
+.experience_list li {
+	margin: 15px;
+	padding: 15px;
+	min-width: 110px;
+	border: 1px solid #ffffff;
+	font-size: 16px;
+	font-family: 'Preahvihear', sans-serif;
```

Comment:
You write this same line inside of a lot of different selectors. Should it actually be applied to the whole document? Or maybe you were having an issue with browser defaults overriding these fonts somehow?

Reply:
> You write this same line inside of a lot of different selectors. Should it actually be applied to the whole document? Or maybe you were having an issue with browser defaults overriding these fonts somehow?

Yes, when I put to body tag = "one font" and to all headers = "another font" after some time while I scrolling page headers fonts are overriding with fonts from body tag.